packages/in_app_purchase/in_app_purchase_android/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 0.3.0+18
 
+* Adds country code api for android.
```

Comment:
nit: specify the api, and also given that this is the android changelog maybe remove "for android"

Something like "Adds new `getCountryCode()` method to `InAppPurchaseAndroidPlatformAddition` to get a customer's country code."